gdx/src/com/badlogic/gdx/graphics/Color.java

```diff
@@ -489,6 +489,94 @@ public static void abgr8888ToColor (Color color, float value) {
 		color.r = ((c & 0x000000ff)) / 255f;
 	}
 
+	/** Sets the RGB Color components using the specified Hue-Saturation-Value. Note that HSV components are voluntary not clamped
+	 * to preserve high range color and can range beyond typical values.
+	 * @param color The Color to be modified.
+	 * @param h The Hue in degree typically from 0 to 360
+	 * @param s The Saturation typically from 0 to 1
+	 * @param v The Value (brightness) typically from 0 to 1
+	 * @return The modified Color for chaining. */
+	public static Color hsvToColor (Color color, float h, float s, float v) {
```

Comment:
Drive-by code review! If it requires a Color to be passed in, why is this static? Why not

    public Color setFromHsv(float h, float s, float v) {
      ...
      return this; // for chaining
    }

Or alternatively:

    public static Color fromHsv(float h, float s, float v) {
      Color color = new Color();
      ...
      return color;
    }

Reply:
Indeed I made that choice to keep consistency of Color class : most of member methods clamp values where static method doesn't.
But I read the class javadoc again and clamping is expected for all color methods (for comparison purpose), so my "high range color" use case may not be valid after all.
I'll probably replace with your proposal and implements clamping. I wait for more comments and core team validation before rework and PR again.
Thanks for the review!

Reply:
For what it's worth: I once needed an unclamped `Color` and just created a new class `UnclampedColor extends Color` and overriding `clamp()` to do nothing. I guess, with none of the methods nor the class itself being `final`, that this is a supported use case, albeit not very explicitly.